examples/github-complete/main.tf

```diff
@@ -125,7 +125,7 @@ resource "random_password" "webhook_secret" {
 
 module "secrets_manager" {
   source  = "terraform-aws-modules/secrets-manager/aws"
-  version = "~> 1.0"
+  version = "1.3.1"
```

Comment:
in examples we don't pin to a specific version, so lets revert changes like this